v2/robotmk/src/bin/scheduler/sessions/schtasks.rs

```diff
@@ -161,6 +163,14 @@ where
     ))
 }
 
+fn start_task(task_name: &str, path_run_flag: &Utf8Path) -> Result<()> {
+    debug!("Starting task {task_name}");
+    write(path_run_flag, "").context(format!("Failed to create run flag file {path_run_flag}"))?;
```

Comment:
Does it really make sense to mix this with the other errors?

Reply:
Not sure, but let's leave it like this for now. Hopefully we anyway find a better approach in the future.